packages/frontend-plugin-api/src/types.ts

```diff
@@ -25,26 +25,20 @@ import { ComponentType, PropsWithChildren } from 'react';
 export type Expand<T> = T extends infer O ? { [K in keyof O]: O[K] } : never;
 
 /** @public */
-export type CoreProgressComponent = ComponentType<PropsWithChildren<{}>>;
+export type CoreProgressProps = PropsWithChildren<{}>;
 
 /** @public */
-export type CoreBootErrorPageComponent = ComponentType<
-  PropsWithChildren<{
-    step: 'load-config' | 'load-chunk';
-    error: Error;
-  }>
->;
+export type CoreBootErrorPageProps = PropsWithChildren<{
```

Comment:
Should I delete this component? It is not used at the moment.

Reply:
Yep let's remove it

Reply:
Done 🙌🏻